src/style.css

```diff
@@ -0,0 +1,112 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+* {
+  margin-top: 0;
+  box-sizing: border-box;
+  font-weight: 300;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+}
+
+.big_form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.big_icon {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  right: 17px;
+  width: 19px;
+  height: 19px;
+  z-index: 1;
+}
+
+.big_input {
+  padding-left: 62px;
+  width: 100%;
+  height: 70px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.small_form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.small_icon {
+  position: absolute;
+  top: 15px;
+  right: 9px;
+  left: 13px;
+  width: 11px;
+  height: 11px;
+  z-index: 1;
+}
+
+.small_input {
+  padding-left: 33px;
+  padding-bottom: 3px;
+  width: 100%;
+  height: 42px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
```

Comment:
Do not duplicate the code, put everything that is repeated in a separate class and add this class to each element where necessary
Fix it everywhere